### PR TITLE
MarkdownBearTest: Inconsistent spacing

### PR DESCRIPTION
**Pull request by akhil117**
Linked to PR: https://github.com/coala/coala-bears/pull/1920
Closes https://github.com/coala/coala-bears/issues/1910

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by naveentata** [comment]
@akhil117  Mention clearly in the shortlog what you actually did like "Remove Inconsistent spacing" 
is better than "Inconsistent spacing"

File: tests/markdown/MarkdownBearTest.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
 from coalib.settings.Setting import Setting
 from coala_utils.ContextManagers import prepare_file
 
+
 test_file1 = """1. abc
 1. def
 """
 
-
```
Comment (Asnelchristian) [inline tests/markdown/MarkdownBearTest.py]: this is okay ... you should not remove it

**Comment by Asnelchristian** [comment]
Nice

**Comment by Asnelchristian** [comment]
Now it is time to rebase, once done :sparkles: :tada:

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1935/commits/115656740be58dacd392f1667c33f29f486bd9f5

**Comment by sils** [comment]
@rultor merge

**Pull request merged**